src/world/partner/goombaria.c

```diff
@@ -8,24 +8,28 @@ void world_goombaria_init(Npc* partner) {
 }
 
 ApiStatus GoombariaTakeOut(Evt* script, s32 isInitialCall) {
-    Npc* owner = script->owner2.npc;
+    Npc* npc = script->owner2.npc;
 
     if (isInitialCall) {
-        partner_init_get_out(owner);
+        partner_init_get_out(npc);
+    }
+    if (partner_get_out(npc)) {
+        return ApiStatus_DONE1;
+    } else {
+        return ApiStatus_BLOCK;
     }
-    return partner_get_out(owner) != 0;
 }
 
 ApiStatus GoombariaUpdate(Evt* script, s32 isInitialCall) {
     PlayerData* playerData = &gPlayerData;
-    Npc* owner = script->owner2.npc;
```

Comment:
What's the reasoning to change these from "owner" to "npc"?

Reply:
I can change it back but i changed them to be consistent with the other functions i was writing. Sometimes the variable would be named owner, other times it would be named npc

Reply:
I think consistently naming them `npc` is better anyway, since `owner` could be an `Actor` or a `Npc` on a first glance

Reply:
I think the meaning of the variable is more important than the type when naming it in a case like this. but it's fine for now regardless

Reply:
If that is the case, then it should be something in the likes of `partnerNpc` or `goombariaNpc` instead of `owner` imo

Reply:
there's no need to specify the type of the variable in its name if there's no ambiguity (like a goombaria Actor* vs a goombaria Npc*) though. so in this case it can just be goombaria

Reply:
My personal preference is to add `Npc` and `Actor` to variable names for clarity sake, but that is just my personal taste and `partner` or `goombaria` are 100% fine with me!